### PR TITLE
Updated the README.md file to describe the repo

### PR DESCRIPTION
**Pull request by Federickg23**

### CODE REVIEW

File: portfolio/src/main/webapp/art.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                    <a href="photography.html">Photos</a>
+                     <a href="profiles.html">Profiles</a>
+                    <a href="martialArt.html">MMA</a>
+                </div> 
+            </div>
+        </div>
+    </div>
+     <div class="bg-text">
+        <h1>Art</h1>
+        <p>Though photography has interested me, I probably would prefer creating my own art as opposed to photographing moments around me. Below are a selection of a few of the pieces I've made. There are several more, though a lot do not have photos, and others are in a notebook that I cannot find. I hope these are enjoyable, and I also hope to be creating more soon!</p>
```
Comment (lincolnv08) [inline portfolio/src/main/webapp/art.html:31]: Split into multiple lines here like

<p>
    ....
    ....
</p>

Comment (Federickg23) [reply portfolio/src/main/webapp/art.html]: Broke down all the longer lines into multiple lines in the submitted files

File: portfolio/src/main/webapp/family.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                    <a href="martialArt.html">MMA</a>
+                </div> 
+            </div>
+        </div>
+    </div>
+    <div style="height:30px;"></div>
+
+    <div class="smallText">
+        <h1>Thank you page</h1>
+        <h3 style="text-align: left;">Welcome! You have found the thank you page! This page is dedicated to my family, who have worked hard to support me. I wouldn't be where I currently am if it wasn't for them, so Thank You! <h3>
```
Comment (lincolnv08) [inline portfolio/src/main/webapp/family.html]: same here!

File: portfolio/src/main/webapp/art.html (hunk 1 above)
Comment (lincolnv08) [inline portfolio/src/main/webapp/art.html]: Split paragraph into multiple lines

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,17 +2,54 @@
...
+     <div class="bg-image" style="background: url('images/teachingBackground.JPG');"></div>
+     <img class="bg-image" src="/images/teachingBackground.JPG">
+     <div class="bg-text">
+        <h1>Federick Gonzalez</h1>
+        <p>Aspiring Programmer, along with other things</p>
+     </div>
+     <h1>About Me</h1>
+        <div class="flex-container">
+            <div class="flex-item" ><img id="square" src="images/romeHeadshot.JPG">
+                <span class="flex-item" > <p class="flex-item"> My name is Federick Gonzalez! I'm currently a rising junior at Colubmia University in the City of New York (And more specificially the FU Foundation School of Engineering and Applied Science). I have been programming since my freshman year of High School, in 2014, and I have learned so much in that span. Whether by picking up Java in AP Computer Science, Python and other languages in Google's CSSI program, or experimenting on my own. I've always enjoyed CS ever since I've discovered it, and hope to futher expand upon my skills!
```
Comment (lincolnv08) [inline portfolio/src/main/webapp/index.html]: Same here and below!

File: portfolio/src/main/webapp/martialArt.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+                    <a href="profiles.html">Profiles</a>
+                </div> 
+            </div>
+        </div>
+    </div>
+    <div style="height:30px;"></div>
+
+    <div class="smallText">
+        <h1>Music and Martial Arts</h1>
+        <h3 style="text-align: left;">Music has been a big part of my life for as long as I can remember. Whether playing my trombone for school, going to orchestra and jazz practice on weekends, or listening to music while working (And even when writing this webpage), music has always been present. Currently, I am a member of the <a href="https://www.youtube.com/channel/UCAaUAFeRINTEPoMOsW3pdfg">Columbia Pops orchestra</a>, and I've included some of our amazing work below. More recently I have found a place as a member of <a href="https://www.youtube.com/user/ColumbiaWushu">Columbia Wushu</a>, a performance based martial arts group, though it is admittedly not my first experience in the realm of martial arts. These two groups make up the communities in which I've found most of my friends, and I'm glad to be a part of them! Below I have some pics and videos so you can have a better idea of what both groups entail!<h3>
```
Comment (lincolnv08) [inline portfolio/src/main/webapp/martialArt.html]: and here :)

File: portfolio/src/main/webapp/photography.html [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+                </div> 
+            </div>
+        </div>
+    </div>
+     <div class="center">
+     <div class="bg-image" style="background: url('images/photography.JPG');"></div>
+     <img class="bg-image" src="/images/photography.JPG">
+     <div class="bg-text">
+        <h1>Photography</h1>
+        <p>Photography has always been interesting for me, though I have definitely not always been the best at it. The ability to capture photos is a useful skill that I plan to do more often. Looking back at the places I've been or the friends I've had around never fails to bring back good memories. I thought I'd share some of the locations I took good photos of, in the hopes that they're as enjoyable to you as they are to me.</p>
```
Comment (lincolnv08) [inline portfolio/src/main/webapp/photography.html]: last one!

File: portfolio/src/main/webapp/family.html (hunk 2 above)
Comment (Federickg23) [reply portfolio/src/main/webapp/family.html]: Spacing fixed

File: portfolio/src/main/webapp/art.html (hunk 1 above)
Comment (Federickg23) [reply portfolio/src/main/webapp/art.html]: Spacing here also fixed

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (Federickg23) [reply portfolio/src/main/webapp/index.html]: Yup! This is fixed too

File: portfolio/src/main/webapp/martialArt.html (hunk 4 above)
Comment (Federickg23) [reply portfolio/src/main/webapp/martialArt.html]: Text wrapping is proving to be detrimental to my readability, but it's been corrected!

File: portfolio/src/main/webapp/photography.html (hunk 5 above)
Comment (Federickg23) [reply portfolio/src/main/webapp/photography.html]: Fixed as well!

**Pull request merged**